examples/README.md

```diff
@@ -0,0 +1,17 @@
+# Insert data into GreptimeDB
+
+## Start local GreptimeDB instance via Docker
+
+```shell
+docker run --rm -p 4000-4003:4000-4003 \
+--name greptime greptime/greptimedb standalone start \
+--http-addr 0.0.0.0:4000 \
+--rpc-addr 0.0.0.0:4001 \
+--mysql-addr 0.0.0.0:4002 \
+--postgres-addr 0.0.0.0:4003
+```
+
+## Insert
+
+- [schema](schema/README.md)
+- [tag](tag/README.md)
```

Comment:
I consider that `schema` and `tag` are not good titles.
What does `schema` mean and what does `tag` mean?

Reply:
Good point, what about:

- Table & Schema
- Struct with tag

Reply:
I'm not sure, we use `low-level API` and `ORM API` in the [user documents](https://github.com/GreptimeTeam/docs/pull/803), maybe we need to discuss same names in both user documents and here.